bom-alpha/build.gradle.kts

```diff
@@ -14,4 +14,6 @@ javaPlatform.allowDependencies()
 dependencies {
   // Add dependency on opentelemetry-bom to ensure synchronization between alpha and stable artifacts
   api(platform(project(":bom")))
-}
\ No newline at end of file
+
+  otelBom.addExtra("io.opentelemetry.semconv", "opentelemetry-semconv", "1.21.0-alpha")
```

Comment:
Is it possible to somehow use the version from `dependencyManagement`? Would be nice it it was not possible for them to go out of sync